webapp/src/components/mobilitymap/InstructionsModal.js

```diff
@@ -0,0 +1,246 @@
+import api from "../../api";
+import React from "react";
+import Accordion from "react-bootstrap/Accordion";
+import Alert from "react-bootstrap/Alert";
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+
+const InstructionsModal = () => {
+  const [show, setShow] = React.useState(false);
+  const [accordionKey, setAccordionKey] = React.useState(null);
```

Comment:
Try to avoid changing types (null -> string), It can lead to subtle bugs, and negatively impacts type inference. Use `React.useState('')` instead.